website/docs/components/accordion/partials/specifications/states.md

```diff
@@ -0,0 +1,7 @@
+## States
+
+![Example of accordion states](/assets/components/accordion/accordion-states.png =800x*)
```

Comment:
Would it be useful to capture the states of the `isInteractive` variant as well?